indico/modules/events/__init__.py

```diff
@@ -256,3 +257,31 @@ def _event_cloned(old_event, new_event, **kwargs):
 @template_hook('event-references-list')
 def _inject_event_references(event, **kwargs):
     return render_template('events/management/reference_list.html', event=event, references=event.references, **kwargs)
+
+
+@template_hook('event-location-field')
+def _render_location_field(event, **kwargs):
+    from indico.modules.events.forms import EventLocationForm
+
```

Comment:
no need for that empty line. for local imports in small functions it doesn't improve readability